### ISSUE TITLE
TDEE calculation

### ISSUE THREAD

**Issue by Mitlik** [davidhealey/waistline#56]
Assuming both weight and food are added to the program it has everything required to do a total daily energy expendature (TDEE) calculation:  ((m_1 - m_0)*0.009 + calories)/#days

I'm not sure of where it would fit into the UI though. Maybe a separate page with a date range selector.

**Comment by davidhealey** [comment]
Yes I probably will add this at some point. I'll put it in the goals section as it will be used to calculate a calorie target.
